CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# v1.27.0
+### Notable Changes
+* Pre-stop Lifecycle Hook enhancements ([#1895](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1895), [@torredil](https://github.com/torredil))
+
+### Bug Fixes
+* Fix taint removal retry for non-swallowed errors ([#1898](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1898), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
+* Remember likely bad device names for an hour ([#1889](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1889), [@ConnorJC3](https://github.com/ConnorJC3))
```

Comment:
I know what you wrote is the name of the PR, but can you add something like `(Enable use of driver on AMIs with instance store mounts)`, as that's the customer facing change of this PR